config/fontawesome.js

```diff
@@ -0,0 +1,25 @@
+const _ = require('underscore');
+const camelCase = require('lodash.camelcase');
+
+const { icon, config } = require('@fortawesome/fontawesome-svg-core');
+const fasIcons = require('@fortawesome/pro-solid-svg-icons');
+const farIcons = require('@fortawesome/pro-regular-svg-icons');
+const falIcons = require('@fortawesome/pro-light-svg-icons');
+
+config.replacementClass = '';
+
+function getIconHtml(lib, fonts) {
+  return _.map(fonts, iconClass => {
+    const iconName = camelCase(`fa-${ iconClass }`);
+    if (!lib[iconName]) console.error('Missing icon:', iconName);
+    return icon(lib[iconName], { symbol: true }).html;
```

Comment:
@nmajor25 here's the biggest part of the change.  This renders the icon as a symbol during the webpack build rather than for each instance of a template.  The fontawesome libraries are actually not loaded in our app any more other than styles